command/login/login.go

```diff
@@ -109,6 +109,10 @@ func (c *cmd) bearerTokenLogin() int {
 			c.UI.Error(fmt.Sprintf("Error with aws-iam auth method: %s", err))
 			return 1
 		} else {
+			if c.aws.outputToken {
```

Comment:
The flag validation could use an update, so that e.g. the -token-sink-file flag and certain other fields aren't required (because it will not make the login request). We could also consider moving this functionality to another command, or etc 🤷